src/components/attendee/SearchOverlay/SpaceItem/SpaceItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+import { Button } from "components/attendee/Button";
+
+import { SPACE_TAXON } from "settings";
+
+import { SpaceWithId } from "types/id";
+
+import { enterSpace } from "utils/url";
+
+import { useWorldById } from "hooks/worlds/useWorldById";
+
+import CN from "../SearchOverlay.module.scss";
+
+export type SpaceItemProps = {
+  space: SpaceWithId;
+  onClick: () => void;
+};
+
+export const SpaceItem: React.FC<SpaceItemProps> = ({ space, onClick }) => {
+  const { world } = useWorldById({ worldId: space.worldId });
+
+  const { push: openUrlUsingRouter } = useHistory();
+
+  const handleEnterSpace = useCallback(() => {
+    enterSpace(world?.slug, space?.slug, {
+      customOpenRelativeUrl: openUrlUsingRouter,
+    });
+    onClick();
+  }, [world?.slug, openUrlUsingRouter, onClick, space?.slug]);
+
+  return (
+    <div>
+      <div className={CN.searchItemResultHeader}>
+        <h3 className={CN.searchItemResultTitle}>
+          {space.name}
+          <span>{SPACE_TAXON.title}</span>
+        </h3>
+      </div>
+      {space.subtitle && (
+        <p className={CN.searchItemResultSubtitle}>{space.subtitle}</p>
+      )}
```

Comment:
do we edit anywhere in the admin these subtitles? I don't think so... I guess we should better remove that.

Reply:
There's a `subtitle` field for any kind of space, so I think we should keep it 
<img width="379" alt="image" src="https://user-images.githubusercontent.com/56254806/167825324-a0fca681-68f0-49f1-8b17-6dc723999ac1.png">

Reply:
But this subtitle is saved also in the config if I'm not mistaken
```
space.config?.landingPageConfig?.subtitle
```